### PR TITLE
Revenants now have randomly generated flavor names

### PR DESCRIPTION
**Pull request by Arkatos1**
**What does this PR do:**
When revenant is spawned, he is now also randomly generated a flavor name. This name looks like this:
[spirit_type] + "of" + [adjective] + [theme]. Adjective has also a small chance to be skipped. I have stated some examples below, but there are many combinations. You can see all possible words at strings/revenant_names.json in the Files. Feel free to suggest some, but they have to appropriate to the  general revenant theme. 

**Example names:**
Spirit of Hate
Revenant of Silent Death
Phantom of Brutal Grief
Spectre of Sorrow
Soul of Indifferent Desperation

Ported from /tg/station13.

Tested locally without any issue.

**Changelog:**
:cl: Arkatos
add: Revenants now have randomly generated flavor names
/:cl:

### CODE REVIEW

File: strings/revenant_names.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "Spirit",
+        "Ghost",
+        "Spectre",
+        "Phantom",
+        "Revenant",
+        "Essence",
+        "Soul"
+    ],
+
+    "adverb": [
```
Comment (Tayyyyyyy) [inline strings/revenant_names.json]: These are actually adjectives 😮

**Comment by taukausanake** [comment]
I can already see some delicious names coming from this, and I love it

File: strings/revenant_names.json (hunk 1 above)
Comment (taukausanake) [reply strings/revenant_names.json]: Can we add "hilarious" to this list? You have "laughter" as a possible theme. inb4 clown ghost

Comment (Arkatos1) [reply strings/revenant_names.json]: Adverbs turned into adjectives, thanks. Hilarious added as a possible adjective, we have one "lighter" theme, so I guess it alright if we have also one lighter adjective.

**Comment by Mitchs98** [comment]
The latter parts of the name should be capitalized imo. I.E 

Spirit of Grief

Otherwise seems nice....just a grammar nitpick >_>

**Comment by Keilden** [comment]
Needs salt as theme to fit in with the theme of ghosts on SS13.

**Comment by TDSSS** [comment]
Gonna make it harder to follow the rev as ghost, but nice flavour addition. Could probably do the same for slaughter demons.

File: strings/revenant_names.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	"fiery ",
+	"silent ",
+	"violent ",
+	"peaceful ",
+	"indifferent ",
+	"hilarious ",
+	"brutal "
+    ],
+
+    "theme": [
```
Comment (Citinited) [inline strings/revenant_names.json]: Why is salt not a possible theme 🤔

File: strings/revenant_names.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        "Ghost",
+        "Spectre",
+        "Phantom",
+        "Revenant",
+        "Essence",
+        "Poltergeist",
+        "Soul"
+    ],
+
+    "adjective": [
```
Comment (taukausanake) [inline strings/revenant_names.json]: Let's get "spiteful" as a choice too. Also, can you sort the options by alphabetical order? Would make it easier to read for those of us with shit brains

File: strings/revenant_names.json (hunk 2 above)
Comment (Arkatos1) [reply strings/revenant_names.json]: Salt added. Literally.

File: strings/revenant_names.json (hunk 3 above)
Comment (Arkatos1) [reply strings/revenant_names.json]: Spiteful added. All lists were alphabetically sorted.

**Comment by Arkatos1** [comment]
@Mitchs98 Capitalization added. You were right, looks much better.

@Keilden Salt added. Literally.

**Comment by Mitchs98** [comment]
Perfecto.

**Pull request merged**